notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-989_analysis_inertia_compensation.ipynb

```diff
@@ -0,0 +1,699 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #101.            tma_el["timestamp"] = pd.to_datetime(tma_el["timestamp"], unit="s") - tai_utc_offset 
</pre><p>Might be better to write as Time(tma_az["timestamp"], format="unix_tai", scale="utc").datetime and let astropy handle the tai_utc_offset</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>